Content/LeagueSandbox-Scripts/Champions/Sion/Passive.cs

```diff
@@ -0,0 +1,72 @@
+using GameServerCore.Domain.GameObjects;
+using LeagueSandbox.GameServer.Scripting.CSharp;
+using GameServerCore.Domain.GameObjects.Spell;
+using GameServerCore.Domain.GameObjects.Spell.Missile;
+using System.Numerics;
+using GameServerCore.Scripting.CSharp;
+using LeagueSandbox.GameServer.API;
+using GameServerCore.Domain;
+using GameServerLib.GameObjects.AttackableUnits;
+using static LeagueSandbox.GameServer.API.ApiFunctionManager;
+
+
+namespace Spells
+{
+    public class SionQ : ISpellScript //Was supposed to be "SionPassive", but passive scripts aren't getting read 
```

Comment:
"Was supposed to be "SionPassive", but passive scripts aren't getting read" - Is it a bug that passives are not being read? If yes, please add "refer to issue #1183. TODO: fix it".